clicommand/redactor_add.go

```diff
@@ -52,9 +52,26 @@ type RedactorAddConfig struct {
 }
 
 var RedactorAddCommand = cli.Command{
-	Name:        "add",
-	Usage:       "Add values to redact from a job's log output",
-	Description: "This may be used to parse a file for values to redact from a running job's log output. If you dynamically fetch secrets during a job, it is recommended that you use this command to ensure they will be redacted from subsequent logs. Secrets fetched with the builtin ′secret get′ command do not require the use of this command, they will be redacted automatically.",
+	Name:  "add",
+	Usage: "Add values to redact from a job's log output",
+	Description: `Usage:
+
+    buildkite-agent redactor add [options...]
+
+  Description:
+
+  This command may be used to parse a file for values to redact from a
+	running job's log output. If you dynamically fetch secrets during a job,
+	it is recommended that you use this command to ensure they will be
+	redacted from subsequent logs. Secrets fetched with the builtin
+	′secret get′ command do not require the use of this command, they will
+	be redacted automatically.
+
+  Example:
+
+    $ buildkite-agent redactor add ...
+
+	Redacts something.`,
```

Comment:
Hello @wolfeidau ,

Could you (or would you know anyone who could) please provide an example of how this command is used? Just a typical example using some of the options would be great, and then I can provide a better explanation than what's currently provided in the agent.

Reply:
FYI, I fixed the spacing in the Description section only, which is why this comment appears outdated, but it's actually not.